exasol/nb_connector/cli/options/all.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
```

Comment:
I find the name of the file a bit confusing. "ould "common.py" be a better name?